src/index.js

```diff
@@ -304,6 +307,21 @@ Machine.prototype.snapshots = function () {
     };
 };
 
+Machine.prototype.boxRepackage = function (name, provider, version, cb) {
+    if (typeof name !== 'string') {
+        cb('name must be provided as a string');
+    }
+    if (typeof provider !== 'string') {
+        cb('provider must be provided as a string');
+    }
+    if (typeof version !== 'string') {
+        cb('version must be provided as a string');
+    }
+
+    var command = _command(['box', 'repackage', name, provider, version]);
+    this._run(command, cb);
```

Comment:
Do we want to run this line even if the checks on 312, 315, 318 failed and called cb? We'd end up calling the callback twice. I suggest making it return cb(... on those lines.